scripts/quiz-game.js

```diff
@@ -0,0 +1,84 @@
+document.getElementById("start-button").addEventListener("click", quizStarter);
+
+function quizStarter() {
+    // Get player names from inputs
+    const firstPlayer = document.getElementById("firstPlayer").value;
+    const secondPlayer = document.getElementById("secondPlayer").value;
+
+    // Check that the input names are not the same
+    if (firstPlayer === secondPlayer) {
+        alert("Please enter two different names.");
+    }
+    // If the names are not the same then display quiz screen
+    else if (firstPlayer && secondPlayer) {
+        document.getElementById("firstPlayer-name").textContent = firstPlayer;
+        document.getElementById("secondPlayer-name").textContent = secondPlayer;
+
+        document.getElementById("start-screen").classList.add("hidden");
+        document.getElementById("quiz-screen").classList.remove("hidden");
+    }
+    // If  the inputs are not filled yet
+    else {
+        alert("Please enter names for both players.");
+    }
+}
+document
+    .getElementById("firstPlayer-points")
+    .addEventListener("change", function () {
+        const points = parseInt(this.value);
+        checkWinner("firstPlayer", points);
+    });
+
+document
+    .getElementById("secondPlayer-points")
+    .addEventListener("change", function () {
+        const points = parseInt(this.value);
+        checkWinner("secondPlayer", points);
+    });
+
+function firstPlayerPlus() {
+    const pointsInput = document.getElementById("firstPlayer-points");
+    pointsInput.value = parseInt(pointsInput.value) + 1;
+    checkWinner("firstPlayer-name", pointsInput.value);
+}
+
+function secondPlayerPlus() {
+    const pointsInput = document.getElementById("secondPlayer-points");
+    pointsInput.value = parseInt(pointsInput.value) + 1;
```

Comment:
This works but another way to convert a `string` to a `number` is by simply writing `+pointsInput.value + 1` - the plus in front converts it!

Reply:
such a good idea 👍 I'll use it later